app/src/main/java/com/example/keniphhomemaintenance/navigation/Navigation.kt

```diff
@@ -23,16 +23,21 @@ fun Navigation() {
             DwellingsScreenView(navController = navController)
         }
         composable(
-            route = Screen.MaintenanceListScreen.route + "/{name}" + "/{address}",
+            route = Screen.MaintenanceListScreen.route + "/{id}" + "/{name}" + "/{address}",
             arguments = listOf(
+                navArgument(DwellingsNavArgParameters.DWELLING_ID) {
+                    type = NavType.IntType
+                },
                 navArgument(DwellingsNavArgParameters.DWELLING_NAME) {
                     type = NavType.StringType
-                }, navArgument(DwellingsNavArgParameters.DWELLING_ADDRESS) {
+                },
+                navArgument(DwellingsNavArgParameters.DWELLING_ADDRESS) {
```

Comment:
(comprehension question)
Is it good practice to encapsulate name, address, and ID into DwellingNavArgParameters? I can see how it would clean things up (especially if there are more NavArg parameters in the future), but I'm curious if it's standard practice